aws-cli/bash-linux/s3/bucket-lifecycle-operations/README.md

```diff
@@ -0,0 +1,43 @@
+.. Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+   This work is licensed under a Creative Commons Attribution-NonCommercial-ShareAlike 4.0    International License (the "License"). You may not use this file except in compliance with the    License. A copy of the License is located at http://creativecommons.org/licenses/by-nc-sa/4.0/.
+
+   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
```

Comment:
This copyright section should be a comment so it doesn't show in the rendered version. (markdown uses html comments <!-- this is a comment -->

Reply:
Wrong copyright.

Reply:
Fixed.  Made it an HTML comment and moved the content from below to here.  This is how it reads now:

<!--
All content in this repository, unless otherwise stated, is Copyright © Amazon Web Services, Inc. or its affiliates. All rights reserved.

Except where otherwise noted, all examples in this collection are licensed under the Apache license, version 2.0 (the "License"). The full license text is provided in the LICENSE file accompanying this repository.

   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
-->